#Init Pose Configuration
start_aruco_pose_estimate: true
enable_tf: true
pose_topic: /marker_localization/pose
debug: true

# Image Topic / Camera 
image_topic: /cv_camera/image_raw
marker_publish_topic_name: /found_markers
camera_transform_name: front_camera_optical
camera_reference_tf: base_link
camera_matrix: [554.254691191187, 0.0, 320.50,  0.0, 554.254691191187, 256.50,  0.0, 0.0 ,1.0]
camera_distortion: [0.0, 0.0, 0.0, 0.0, 0.0]

# Aruco Info
marker_length_old: 10
marker_length: [15.0] # Unit: cm
markers_15: [207.0]
# arucoDictionary: 6x6_250 #TODO
marker_name: aruco_6x6_ # the ID will be the last string at the TF. Ex: aruco_6x6_1
aruco_transform_namespace: panelTF/
aruco_reference_tf: /world

# Frequency
publish_frequecy: 40 # Hz